pennylane/math/single_dispatch.py

```diff
@@ -267,11 +267,12 @@ def _take_autograd(tensor, indices, axis=None):
 ar.autoray._FUNC_ALIASES["tensorflow", "arctan2"] = "atan2"
 
 
-def _coerce_tensorflow_diag(x, **kwargs):
-    return ar.autoray.tensorflow_diag(_tf_convert_to_tensor(x), **kwargs)
-
-
-ar.register_function("tensorflow", "diag", _coerce_tensorflow_diag)
+ar.register_function(
+    "tensorflow", "unstack", lambda *args, **kwargs: _i("tf").unstack(*args, **kwargs)
+)
+ar.register_function(
+    "tensorflow", "gather", lambda *args, **kwargs: _i("tf").gather(*args, **kwargs)
+)

```

Comment:
I don't understand why this does not work without manually registering the functions 😕

Reply:
I think autoray is now looking in the experiental module for them.

Reply:
Silly `autoray`

Reply:
I think we may want to do those with 
```python
ar.autoray._SUBMODULE_ALIASES["tensorflow", "unstack"] = "tensorflow"
ar.autoray._SUBMODULE_ALIASES["tensorflow", "gather"] = "tensorflow"
ar.autoray._SUBMODULE_ALIASES["tensorflow", "concat"] = "tensorflow"
```
like about 40 lines above. :) Also seems like we can remove some the `tensorflow.experimental.numpy` registrations up there, now that autoray looks in that module anyways.

Reply:
submodule aliases are workinng, so I'll use that instead.